processing/data_collection/gazette/spiders/mg_itauna.py

```diff
@@ -17,7 +17,7 @@ def parse(self, response):
         last_page = int(
             response.xpath("//select[@id='select']/option[last()]/text()").get()
         )
-        for page in range(1, 1 + last_page):
+        for page in range(2, 1 + last_page):
             yield Request(page_url.format(page=page), callback=self.parse_editions_page)
```

Comment:
Hey @pgarcias01 , right now the spider should get every page but the first one because the `parse` method only "connects" with the `parse_editions_page` method through the requests you are making. You should be able to solve it by adding this:
```suggestion
        for page in range(2, 1 + last_page):
            yield Request(page_url.format(page=page), callback=self.parse_editions_page)

		yield from self.parse_editions_page(response)
```
Since `parse_editions_page` is a generator, this way you can yield everything that is yielded by it "as if it was yielded by `parse` itself". In this case, you would be parsing the first page and yielding the item.

Reply:
Hi @giuliocc done! Thanks, again.

Reply:
Hi @pgarcias01 I'm sorry I made an identation error there (github mobile). The `yield from ...` statement should be outside the `for` loop in this case. Could you fix it? ;/

Reply:
> Hi @pgarcias01 I'm sorry I made an identation error there (it should be correct by now). The `yield from ...` statement should be outside the `for` loop in this case. Could you fix it? ;/

@giuliocc done!